templates/zerver/emails/followup_day1.html

```diff
@@ -26,6 +26,7 @@
     {% endif %}
 
     <p>Thanks,<br/>Zulip</p>
+<img src="{{ realm_uri }}/static/generated/footer.svg"/>
```

Comment:
Do we only want this footer image in this particular email, or should it be in all the emails?